src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -132,6 +128,12 @@ function MoneyRequestView({
     const isTrackExpense = ReportUtils.isTrackExpenseReport(report);
     const {canUseViolations, canUseP2PDistanceRequests} = usePermissions(isTrackExpense ? CONST.IOU.TYPE.TRACK : undefined);
     const moneyRequestReport = parentReport;
+    const linkedTransactionID = useMemo(() => {
+        const originalMessage = parentReportAction && ReportActionsUtils.isMoneyRequestAction(parentReportAction) ? ReportActionsUtils.getOriginalMessage(parentReportAction) : undefined;
+        return originalMessage?.IOUTransactionID ?? '-1';
+    }, [parentReportAction]);
+    const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${linkedTransactionID}`);
```

Comment:
NAB: To match the previous behavior using withOnyx should we render null if the transaction status is loading from Onyx?